File: C++/1.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        map<int, int> mymap;
        for(int i=0;i < nums.size();i++){
            if(mymap.find(nums[i]) != mymap.end()){
                return {mymap[nums[i]],i+1};
            }
            mymap[target - nums[i]] = i + 1;
        }
    }
};

File: LeetCodePython/1.py
class Solution:
    # @param {integer[]} nums
    # @param {integer} target
    # @return {integer[]}
    def twoSum(self, nums, target):
        dic = {}
        for i in range(len(nums)):
            if nums[i] in dic:
                return [ dic[nums[i]], i+1]
            else:
                dic[target - nums[i]] = i+1